src/main.c

```diff
@@ -87,18 +87,18 @@ static msg_t nav_msg_thread(void *arg)
         __asm__("CPSIE i;");
 
         if (ret < 0) {
-          printf("PRN %02d ret %d\n", tracking_channel[i].prn+1, ret);
+          log_info("PRN %02d ret %d\n", tracking_channel[i].prn+1, ret);

```

Comment:
Yeah, so I remember seeing this before in the logs and being confused. Would it be possible to add some additional context here, such as the meaning of this return value and context of the statement?